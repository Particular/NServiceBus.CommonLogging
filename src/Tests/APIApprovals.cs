using System.Reflection;
using NUnit.Framework;
using Particular.Approvals;
using PublicApiGenerator;

[TestFixture]
public class APIApprovals
{
    [Test]
    public void Approve()
    {
        var publicApi = ApiGenerator.GeneratePublicApi(Assembly.Load("NServiceBus.CommonLogging"), excludeAttributes: new[] { "System.Runtime.Versioning.TargetFrameworkAttribute" });
        Approver.Verify(publicApi);
    }
}