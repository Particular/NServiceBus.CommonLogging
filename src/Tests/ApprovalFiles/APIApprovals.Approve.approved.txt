namespace NServiceBus
{
    public class CommonLoggingFactory : NServiceBus.Logging.LoggingFactoryDefinition
    {
        public CommonLoggingFactory() { }
        protected override NServiceBus.Logging.ILoggerFactory GetLoggingFactory() { }
    }
}